backend/wmg/data/load.py

```diff
@@ -0,0 +1,19 @@
+def update_s3_resources():
```

Comment:
will this call the below methods? could stub that out now, if desired

Reply:
it will call the below methods and copy the data stored under the name of the cube to s3 under the snapshot id